src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferAgent.java

```diff
@@ -1903,7 +1905,9 @@ private static void pushFileToRemoteStore(
       int parallel,
       File srcFile,
       boolean uploadFromStream,
-      RemoteStoreFileEncryptionMaterial encMat)
+      RemoteStoreFileEncryptionMaterial encMat,
+      String streamingIngestClientName,
+      String streamingIngestClientKey)
```

Comment:
Might it make things more flexible to accept a Map of `<String, String>` and then enumerate keys to add metadata? Seems like a nice future-proof measure.